packs/bestiary-effects/effect-bosuns-command.json

```diff
@@ -0,0 +1,79 @@
+{
+    "_id": "WfiaKdXNSxC3POcs",
+    "img": "systems/pf2e/icons/spells/agitate.webp",
+    "name": "Effect: Bosun's Command",
+    "system": {
+        "description": {
+            "value": "<p>The creature gains the bosun's choice of a +2 status bonus to attack rolls or <span style=\"font-family:Roboto, sans-serif\">a +10 status bonus its Speeds.</span></p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 3
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Gamemastery Guide"
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.AttackLabel",
+                        "value": "attack"
+                    },
+                    {
+                        "label": "PF2E.Actor.Speed.Label",
+                        "value": "speed"
+                    }
+                ],
+                "flag": "bossunCommand",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.BossunsCommand"
+            },
+            {
+                "key": "RollOption",
```

Comment:
Consider using `"rollOption"` on the ChoiceSet instead.

Reply:
Do you have an example I can follow?

Reply:
Take a look at Able Carter.

Reply:
Thank you!